sbot/robot.py

```diff
@@ -144,3 +145,7 @@ def wait_start(self) -> None:
         LOGGER.info("Waiting for start button.")
         self.power_board.piezo.buzz(timedelta(seconds=0.1), Note.A6)
         self.power_board.wait_for_start_flash()
+        LOGGER.info("Start button pressed.")
+
+        if self.is_competition:
+            kill_after_delay(180)
```

Comment:
Maybe make the magic number a constant in the game metadata?

Reply:
0ad82cf